packages/addon-dev/src/rollup-public-assets.ts

```diff
@@ -51,7 +51,7 @@ export default function publicAssets(
       });
       const publicAssets: Record<string, string> = filenames.reduce(
         (acc: Record<string, string>, v): Record<string, string> => {
-          const namespace = opts?.namespace ?? join(pkg.name, path);
+          const namespace = opts?.namespace ?? pkg.name;
```

Comment:
doesn't this mean that all the public assets will just be flattened 🤔 you can't have any structure at all any more

Reply:
@mansone No, this is just the namespace, that is statically added to any file that is within the assets folder. Let's say the package name is `fancy-addon`, and its rollup config has `addon.publicAssets('src/assets/public')`, and it has a file in `src/assets/public/foo/bar.jpg`. Then before this change the plugin would add this to the package.json:

```
"public-assets": {
  "./src/assets/public/foo/bar.jpg": "fancy-addon/src/assets/public/foo/bar.jpg"
}
```

With this PR it would be:
```
"public-assets": {
  "./src/assets/public/foo/bar.jpg": "fancy-addon/foo/bar.jpg"
}
```

`path` in L54 would be `src/assets/public` in this example (as passed to the plugin), so that gets removed here.